app/upgrades_test.go

```diff
@@ -421,3 +429,210 @@ func (s *UpgradeTestSuite) TestRemoveP8eMemorializeContractFee() {
 		})
 	}
 }
+
+func (s *UpgradeTestSuite) TestRemoveInactiveValidatorDelegations() {
```

Comment:
The `TestRust` and `TestRustRc1` should each have an expected line added to them: `"INF removing all delegations from validators that have been inactive (unbonded) for {whatever} days".